### DISCUSSION TITLE
How can I get co-authors from Github REST API?

### DISCUSSION THREAD

**Discussion by zion830** [community/community#30313] [Apps, API and Webhooks] [answered]
Is there a way to get the co-authors information?
I can't find in [get-a-commit](https://docs.github.com/en/rest/commits/commits#get-a-commit) api document.

**Comment by airtower-luna** [comment, marked as answer]
I'm not aware of any dedicated API either, but you can look a the `.commit.message` element of the response and check if there are any `Co-authored-by:` trailers.

**Reply by zion830** [reply]
thanks for letting me know! 😄

**Reply by airtower-luna** [reply]
Glad I could help! Marking the answer would be appreciated. :smile_cat:
